locales/en/create-kafka-instance.json

```diff
@@ -16,7 +16,7 @@
     "form_invalid": "Address form errors to proceed.",
     "over_quota_message": "Your organization reached the limit of $t(common:kafka) instances available for creation.",
     "over_quota_title": "$t(common:kafka) instance can't be created",
-    "trial_unavailable_message": "$t(common:kafka) instances are no longer available. Try again in a few hours.",
+    "trial_unavailable_message": "All available $t(common:kafka) evaluation slots are currently in use. Please try again later",
```

Comment:
I'm confused.  What is the another key to use?

Reply:
So, we have two moments where we show an alert/error to the user for trial instances unavailable.

One is when a user opens the dialog, there is capacity, then fills all the info and hits submit. If the availability changed while doing this, we show an error message. That message is baked by the key you changed.

There is another case tho, which is when there is no capacity when the user opens the dialog. When that happens, we show a warning message with this as the copy https://github.com/redhat-developer/app-services-ui-components/pull/227/files#diff-6dc2a291b505ae5039da98af4d38441d9fcd38f894b6a6f49a850d356cb7e8d6R56

Reply:
The message body we want to show is the same for both cases. Should we use the same key—`"trial_unavailable_message"`—for both cases? 

Is there ever a possibility where we would need a key like `"instance_or_region_unavailable_message"` for trial users? If we have limited trial instances available, I assume we'd have regions available to support that limited number. And even if we are out of regions for creating trial instances, should we still use the same `"trial_unavailable_message"` message for this case?  

For standard users, I'm proposing that we modify the message we display when all regions are unavailable to something like this based on recent UX review of the creation flow:
> **Cloud provider regions are temporarily unavailable.**
> Try again later or contact support.

Reply:
Here are my suggestions for the message:
1. Don't say "eval", but say "trial" instead. This is based on some of the notes in [this epic brief regarding updates to trials](https://docs.google.com/document/d/1RxTRpjgA7t6OSBRBI7MjmMA21cvFCBbs6VZuRqPI6o4/edit#)
2. Don't say "Please". There's probably some explanation available for this in the style guide which I don't have currently. 
3. This is my own personal preference—"slot" does not seem like a user-facing term.
4. Another personal preference—we state "Kafka" in the message title, so I think we can drop it from the message body.
5. Was there an explicit point made about "try again in a few hours" that prompted a change to "try again later"? The former was recommended by our content strategist at the time to provide some time frame that is less ambiguous. This was based on the assumption that as trial instances expire, there could be instances that become available in the next few hours. I did read the email thread, but it seemed like the issue was that there was too much ambiguity regarding why an instance can't be created. I'd suggest keeping the original phrase.  

Based on that, I'd suggest using this for the message body:
> "All available trial instances are currently in use. Try again later.",

However, I would like to run this by content strategists to get their input on this before making this update.

Reply:
Fully agree. Please update us once that is done

Reply:
I heard back from the content designer, and she had no suggestions to make. This is the message we should display if no trial instances are available for creation:
> All available trial instances are currently in use. Try again later.

The question I still have is whether we display this message for the trial user if no regions are available, too?

For standard users, we want to display the following message as part of the next set of design updates we're working on. I don't think this makes sense to display for trial users, because if a region is unavailable, doesn't that mean that all trial instances are in use? But I'm not sure if that is true.

> Cloud provider regions are temporarily unavailable.
> Try again later or contact support.

Reply:
> The question I still have is whether we display this message for the trial user if no regions are available, too?

Yes, [this is what we display in that case](https://redhat-developer.github.io/app-services-ui-components/?path=/story/kafka-create-kafka-instance-create-kafka-instance--no-regions-returned-from-api-on-form-load) in that case. It doesn't make any distinction between trial users or standard users. Actually, we don't reason in terms of trial or standard users at all, we have just users that can create standard instances or trial instances. Hope it makes sense.

Reply:
So @jgiardino I'm not sure I understand what the next step should be. Can I propose scheduling a call to discuss this offline? It might be more effective than commenting here.